Orange/widgets/tests/test_workflows.py

```diff
@@ -0,0 +1,33 @@
+from os import listdir
+from os.path import isfile, join, dirname, exists
+
+from Orange.canvas.application import workflows
+from Orange.canvas.scheme import widgetsscheme
+from Orange.canvas.scheme.readwrite import scheme_load
+from Orange.widgets.tests.base import WidgetTest
+from Orange.widgets import tests
+
+class TestWorkflows(WidgetTest):
+    def test_scheme_examples(self):
+        """
+        Test if Orange workflow examples can be opened. Examples in canvas
+        and also those placed "workflows" subfolder.
+        GH-2240
+        """
+        def test(filename):
+            new_scheme = widgetsscheme.WidgetsScheme()
+            with open(filename, "rb") as f:
+                scheme_load(new_scheme, f)
+
+        tutors = workflows.example_workflows()
```

Comment:
Move the specification of the workflows to be tested outside of the tests. I define them in a constant on a module level that would look like this:
```
TEST_WORKFLOWS=itertools.chain(
    workflows.example_workflows(),
    discover_workflows(dirname(__file__))
)
```
(I assume that you write a function discover_workflows that gets a path to a dir and yields all .ows files it finds by traversing the directory) This would simplify the test to the point where the checks in test() function could be written in the body of the for loop that iterates through the TEST_WORKFLOWS.